dub.sdl

```diff
@@ -41,9 +41,6 @@ configuration "vibe-core" {
 configuration "win32_mscoff" {
 	subConfiguration "vibe-d:core" "win32_mscoff"
 }
-configuration "libevent" {
-	subConfiguration "vibe-d:core" "libevent"
-}
```

Comment:
I'd still leave this and just remove the whole :core sub package as part of 0.10.0.